docs/reference_keymap_extras.md

```diff
@@ -1,86 +1,91 @@
 # Language-specific Keycodes
 
-Keyboards are able to support a wide range of languages. However, they do not send the actual characters produced by pressing their keys - instead, they send numerical codes. In the USB HID spec, these are called "usages", although they are more often referred to as "scancodes" or "keycodes" when in the context of keyboards.
-Less than 256 usages are defined in the HID Keyboard/Keypad usage page, and some of those do nothing on modern operating systems. So, how is this language support achieved?
+Keyboards are able to support a wide range of languages. However, this support is not actually achieved within the keyboard itself - instead, it sends numerical codes, which the operating system maps to the appropriate characters based on the user's configured keyboard layout. For example, when a Swedish person presses the key with the `å` character printed on it, the keyboard is *actually* sending the keycode for `[`.
```

Comment:
really, it's not, though.  It's sending a number.  And the host system then interprets that based on the keyboard layout mapping that it uses.  No "character" is really sent. 

And QMK/tmk/etc assumes US English as the default layout.